subprojects/gradle/cthulhu/src/main/kotlin/com/avito/plugin/CthulhuPluginApi.kt

```diff
@@ -0,0 +1,12 @@
+package com.avito.plugin
+
+import com.avito.kotlin.dsl.typedNamed
+import org.gradle.api.tasks.TaskContainer
+import org.gradle.api.tasks.TaskProvider
+
+internal const val markReportAsSourceTaskName = "markReportAsSourceForTMSTask"
+
+fun TaskContainer.markReportAsSourceTask(): TaskProvider<MarkReportAsSourceTask> =
+    typedNamed(markReportAsSourceTaskName)
+
+val cthulhuPluginId = "com.avito.android.cthulhu"
```

Comment:
Can we get pluginId in runtime, instead of duplication here and in `build.gradle.kts` ?
maybe read properties?